node/src/service.rs

```diff
@@ -608,7 +609,13 @@ where
 							slot_duration,
 						);
 
-                        let additional_keys = vec![(para_head_key, para_head_data), (relay_slot_key, Slot::from(u64::from(*relay_slot)).encode())];
+                        let current_block_randomness = BlakeTwo256::hash(timestamp.encode().as_slice());
+
+                        let additional_keys = vec![
+                            (para_head_key, para_head_data),
+                            (relay_slot_key, Slot::from(u64::from(*relay_slot)).encode()),
```

Comment:
Is there a reason why we need to include these keys? I thought we would only need to add  `current_block_randomness_key`.

Reply:
Those were there before

Reply:
oh yes I remember now